ctk/common/src/main/kotlin/org/codice/compliance/verification/core/CommonDataTypeVerifier.kt

```diff
@@ -60,7 +60,7 @@ class CommonDataTypeVerifier {
 
         /** 1.3.1 String Values **/
         @Suppress("StringLiteralDuplication")
-        fun verifyStringValues(node: Node, errorCode: SAMLSpecRefMessage?) {
+        fun verifyStringValues(node: Node, errorCode: SAMLSpecRefMessage? = null) {
```

Comment:
Changes for `CommonDataTypeVerifier` will be https://github.com/connexta/saml-conformance/pull/96